DTM content for release 10.17 (#861)
SoftwareAG/c8y-docs: content/dtm/asset-hierarchy-bundle/viewing-assets.md

In the following dialog box, click **Confirm** to continue.
Optionally select the checkbox in the dialog box to delete all subassets and devices for the selected asset.

### To search for an asset

You can search for assets through the **Search** button at the right of the top bar.

> On entering a search term into the textbox at the top of the Search window, the application returns all assets matching the search criteria.

Reviewer: ```suggestion
Enter a search term into the textbox at the top of the **Search** window to see all assets matching the search criteria on the page **Search results**.
```

> Under **Search results**, the assets matching the search criteria are shown. To see more details click **Go to the asset data table** at the bottom right. This will show the entire search results in a table format.

Reviewer: combine the first sentence with the one above (see comment above).

Author: Removed `Under **Search results**, the assets matching the search criteria are shown.` as it is combined in the previous paragraph.

> The Search window only lists a limited number of matches. In case of more matches, to see the complete results you must switch to the asset data table.

Reviewer: ```suggestion
The **Search** window only shows a limited number of matches. In case of more matches, switch to the asset data table to see the complete results.
```

![assets-search](/images/dtm/assets/dtm-assets-search-assets.png)

{{< c8y-admon-important >}}

> The search results include all assets containing the search term in any property (name, model or any fragment), that means, the search results do not only include assets matching the search criteria with their names.

Reviewer: ```suggestion
The search results include all assets containing the search term in any property (name, model or any fragment), that is, the search results do not only include assets matching the search criteria with their names.
```
